diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter1/PackageNamingWarnTest.kt

```diff
@@ -71,6 +71,24 @@ class PackageNamingWarnTest : LintTestBase(::PackageNaming) {
         )
     }
 
+    @Test
+    @Tag(WarningNames.PACKAGE_NAME_MISSING)
+    fun `don't add the package name to the file in buildSrc path`() {
+        lintMethod(
+            """
+                import org.cqfn.diktat.a.b.c
```

Comment:
We should check that if there is no package name, we don't trigger a warning. If package name is present and is incorrect, other warnings should still be activated

Reply:
Oh, that's `iport`, not `package`, sorry 👀